libs/blocks/icon-block/icon-block.css

```diff
@@ -1,3 +1,8 @@
+ :root {
+  /* border-radius sizes */
+  --l-rounded-corners: 16px;
+}
```

Comment:
Please scope this to the block, not to `:root`. Should this value change, it will interfere with the values we have in the more global file: https://github.com/adobecom/milo/blob/stage/libs/styles/rounded-corners.css#L1-L7